cmd/oci/oci.go

```diff
@@ -218,6 +219,15 @@ var OCICmd = &cobra.Command{
 				os.Exit(1)
 			}
 			fmt.Println(styles.SucessStyle.Render(fmt.Sprintf("Image %s pushed to registry", artifact.Name)))
+
+			if digest {
```

Comment:
Won't this be save along with other output? Should we be saving it in a file instead?